src/point_one/fusion_engine/messages/measurements.h

```diff
@@ -1156,9 +1156,10 @@ struct alignas(4) RawHeadingOutput : public MessagePayload {
  *        version 1.0).
  * @ingroup measurement_messages
  *
- * The measurements in this message have been corrected for sensor mounting
- * angle biases.
- *
+ * The HeadingOutput message behaves similarly to the RawHeadingOutput,
```

Comment:
Make sure to add `@ref` for all Doxygen references (including `@ref SolutionType::Invalid` below)